### PR TITLE
Frontend - improving website style

### PR DESCRIPTION
**Pull request by karenav**
In this branch we pulled the unmerged branch "sendInput", in order to work with the updated fronend logic, and that's why we see in this PR the commits of that branch. The *only* relevant commits are "improve frontend style" ef17375 and the lint fixes after it.

### CODE REVIEW

**Review by tbarnahor — APPROVED** [review]

**Comment by tbarnahor** [comment]
Karen and I worked on this PR together, so I didn't add any review comments.
We will be happy to get your review :)

File: project/src/main/webapp/script.js [hunk 1]
```diff
@@ -111,18 +104,32 @@ function getUserLocationFromUi() {
...
+function displayNumResultsMessage(numResults) {
+  let messageElement = document.getElementById('problem-message-container');
+  const tryAgainMessage =
+      'You are welcome to try again, and maybe try to change some of the entered parameters.';
+  if (numResults === 0) {
+    messageElement.innerHTML = 'Your search had no results. ' + '<br>' + tryAgainMessage
+  } else if (numResults === 1) {
+    messageElement.innerHTML = 'Your search had only 1 result. ' + '<br>' + tryAgainMessage
+  } else if (numResults === 2) {
+    messageElement.innerHTML = 'Your search had only 2 results. ' + '<br>' + tryAgainMessage
```
Comment (szwbar) [inline project/src/main/webapp/script.js]: Include the \<br\> as part of the `tryAgainMessage`?

**Review by szwbar — APPROVED** [review]
Looks good, thanks for cleaning up the code as well!

File: project/src/main/webapp/script.js (hunk 1 above)
Comment (karenav) [reply project/src/main/webapp/script.js]: Done

**Pull request merged**